data/json/bionics.json

```diff
@@ -455,24 +739,41 @@
         "type": "bionic",
         "id": "bio_blaster",
         "name": "Fusion Blaster Arm",
+        "occupied_bodyparts": [
+            [ "ARM_L", 20 ],
+            [ "HAND_L", 5 ]
+        ],
         "act_cost": 50,
         "description": "Your left arm has been replaced by a heavy-duty fusion blaster!  You may use your energy banks to fire a damaging heat ray; however, you are unable to use or carry two-handed items, and may only fire handguns."
     },{
         "type": "bionic",
         "id": "bio_laser",
         "name": "Finger-Mounted Laser",
+        "occupied_bodyparts": [
+            [ "HAND_R", 1 ]
+        ],
         "act_cost": 50,
         "description": "One of your fingers has a small high-powered laser embedded in it.  This long range weapon is not incredibly damaging, but is very accurate, and has the potential to start fires."
     },{
         "type": "bionic",
         "id": "bio_emp",
         "name": "Directional EMP",
+        "occupied_bodyparts": [
+            [ "ARM_R", 1 ],
+            [ "HAND_R", 2 ]
+        ],
         "act_cost": 100,
         "description": "Mounted in the palms of your hand are small parabolic EMP field generators.  You may use power to fire a short-ranged blast which will disable electronics and robots."
```

Comment:
```palms of your hand``` - does it mean "palm of your hand" (as in Water Extraction Unit below) or "palms of your hands"?